docs/userguide/layout.md

```diff
@@ -1,293 +1,296 @@
 # Layout
-The most prominent objects in Elsie are boxes. They serve as a layout mechanism
-and the way how to put a content into a slide.
-
-### Creating boxes
-
-Let us create the following example, where 3 boxes are created from the
-top-level box. The new box is created by calling `.box(...)` on an existing box.
-The parameter ``slide`` passed into the slide function is also a box.
-
-```python
-@elsie.slide()
-def boxdemo1(slide):
+*Elsie* contains a layout system which allows you to quickly build scenarios that are common in
+presentations, while still providing the option of finely tuned customization for situations where
+every pixel placement matters.
+
+The central element of the layout system is the **Box**.
+
+Each *Elsie* slide contains a layout hierarchy tree. The internal nodes of the tree are *boxes* and
+the leaves are *box items*. Boxes are layout containers, which do not produce any visual content, but
+they dictate how are their children laid out on a slide. Box items are individual paintable items,
+such as text, images, shapes, etc. Anything that can be rendered by *Elsie* thus has an accompanied
+parent box which decides its size and position on a slide.
+
+## Creating boxes
+To create a new box, you can call the [`box`](elsie.boxmixin.BoxMixin.box) method on an existing
+[`Box`](elsie.boxmixin.BoxMixin). This will return a new box which will be a child of the box object
```

Comment:
There is actually a class Box, so maybe this should lead to Box class rather than BoxMixin? (Or should we explain box mixin?)

Reply:
Maybe we can just remove the link?

Reply:
Removing the link is maybe the less confusing. But I do not know